packages/noya-designsystem/src/utils/sketchPattern.ts

```diff
@@ -0,0 +1,69 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import { FileMap } from 'noya-sketch-file';
+
+export const imgFileExtensions = ['png', 'jpeg', 'webp'];
+export const mimeTypes = imgFileExtensions.map(
+  (e) => `image/${e === 'ico' ? 'x-icon' : e}`,
```

Comment:
I don't think we need to handle `ico` or `x-icon` files. Did you encounter this case?

Reply:
If we just leave it to the extensions we have. I'll remove that part  👍